crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -1058,6 +1069,8 @@ impl SlidingSync {
         // coming from the `OlmMachine::outgoing_requests()` method.
         #[cfg(feature = "e2e-encryption")]
         let response = {
+            debug!("Sliding Sync is sending the request _with_ E2EE");
```

Comment:
Not sure if it's clear to you but this is sending the sliding sync request and all the outgoing E2EE related requests out, if any. From the log line it sounds like it's enabling the E2EE extension in the sliding sync request.

Reply:
I've understood it right, but the phrasing of the logs is bad. Rephrased.